src/index.tsx

```diff
@@ -191,9 +193,7 @@ const Routes: React.FC = () => {
                 component={ProductSection}
               />
               <SectionRoute
-                permissions={[
-                  PermissionEnum.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES
-                ]}
+                permissions={[PermissionEnum.MANAGE_PRODUCTS]}
```

Comment:
Should it be changed? I see it is the path to product types, why does it require `MANAGE_PRODUCTS` permission?

Reply:
Looks like faulty rebase